src/Magento/FunctionalTestingFramework/Test/Objects/ActionObject.php

```diff
@@ -496,6 +496,10 @@ private function findAndReplaceReferences($objectHandler, $inputString)
 
             $replacement = $this->resolveParameterization($parameterized, $replacement, $match, $obj);
 
+            if (is_array($replacement)) {
+                $replacement = '["' . implode('","', $replacement) . '"]';
```

Comment:
We should array_walk/array_map with `addSlashes` to break quotes at the very least. Inadvertend variable interpolation in MFTF is not yet handled, so this is not introducing anything new.